server/events/vcs/common/common.go

```diff
@@ -20,17 +18,30 @@ func SplitComment(comment string, maxSize int, sepEnd string, sepStart string) [
 
 	maxWithSep := maxSize - len(sepEnd) - len(sepStart)
 	var comments []string
-	numComments := int(math.Ceil(float64(len(comment)) / float64(maxWithSep)))
-	for i := 0; i < numComments; i++ {
-		upTo := min(len(comment), (i+1)*maxWithSep)
-		portion := comment[i*maxWithSep : upTo]
-		if i < numComments-1 {
-			portion += sepEnd
+	head := 0
```

Comment:
**Explanation:** Split the large comments at LF characters (and not at the middle of the word/line).